package.json

```diff
@@ -66,6 +66,7 @@
     "classnames": "^2.1.2",
     "commonmark": "^0.28.1",
     "counterpart": "^0.18.0",
+    "diff-match-patch": "^1.0.4",
```

Comment:
Is this a big dependency by any chance?

Reply:
It's 75kb of JS.

Reply:
Okay, I assume that's not _too_ much in the scheme of things...